src/OrchardCore/OrchardCore.Queries.Abstractions/IQueryResult.cs

```diff
@@ -0,0 +1,7 @@
+namespace OrchardCore.Queries
+{
+    public interface IQueryResult
+    {
+        public object Items { get; set; }
```

Comment:
Maybe an `IEnumerable<object>` as the results used to be that

Reply:
A SQL Query could return a single value. But I will try.

Reply:
![image](https://user-images.githubusercontent.com/3228637/67052393-0d817400-f10c-11e9-81ff-c59f52ccc3b3.png)
 Works

Reply:
Though I broke something with when we check the option "return documents"